src/en/news/blog/2023/reef-osds-per-nvme/index.md

```diff
@@ -0,0 +1,137 @@
+
+During the Octopus and Pacific development cycles that started changing.  We introduced several performance improvements in the OSD/Bluestore code (For example [here](https://github.com/ceph/ceph/pull/28597) and [here](https://github.com/ceph/ceph/pull/34363)) that greatly improved per-OSD performance.  As Pacific was nearing completion, we ran a variety of tests to determine whether or not our recommendation should change.
+
+![](images/old_FIO.svg)
+
+As expected, Octopus and Pacific were significantly faster than Nautilus, but also no longer showed a consistent performance gain with 2 OSDs per NVMe versus 1.  Some tests still showed some gains, though others showed slight losses.  These tests however are fairly limited in scope.  Does the availability of CPU resournces change the results?  Are there other advantages or disadvantages in terms of performnace or resource consumption?  Finally, we've continued to improve the OSD and bluestore code since Pacific.  Do we still see similar behavior today?  Read on to find out!
+
+# Acknowledgements
+
+First, thank you to [Clyso](https://www.clyso.com) for funding this work to benefit the Ceph community.  Thank you as well to [IBM](https://ibm.com)/[Red Hat](https://redhat.com), [Intel](https://intel.com) and [Samsung](https://samsung.com) for providing the upstream Ceph community with the hardware used for this testing.  Thank you as well to all of the Ceph developers that have worked tirelessly to make Ceph great!
+
+# Cluster Setup 
+
+Nodes | 10 x Dell PowerEdge R6515
+-- | --
+CPU | 1 x AMD EPYC 7742 64C/128T
+Memory | 128GiB DDR4
+Network | 1 x 100GbE Mellanox ConnectX-6
+NVMe | 6 x 4TB Samsung PM983
+OS Version | CentOS Stream release 8
+Ceph Version | Reef v18.2.0 (built from source)
+<br>
+
+5 of the nodes were configured to host OSDs and 5 of the nodes were configured as client nodes.  All nodes are located on the same Juniper QFX5200 switch and connected with a single 100GbE QSFP28 link.  Ceph was deployed and FIO tests were launched using [CBT](https://github.com/ceph/cbt/).  An important OS level optimization on Intel systems is setting the TuneD profile to either "latency-performance" or "network-latency".  This primarily helps by avoiding latency spikes associated with CPU C/P state transitions.  AMD Rome based systems do not appear to be as sensitive in this regard, and I have not confirmed that tuned is actually restricting C/P state transitions, however the tuned profile was nevertheless set to "network-latency" for these tests.
```

Comment:
s/tuned/TuneD/  (3 times)
FWIW on CentOS7 Dell R640s with Xeons I've verified with `turbostat` that these profiles do restrict C-state transitions.

Reply:
Yes, I should clarify here that I mean specifically for AMD processors.